content/concepts/introduction/software-ecosystem.md

```diff
@@ -8,6 +8,7 @@ labels:
 # Software Ecosystem
 
 The XRP Ledger is home to a deep, layered ecosystem of software projects powering and enabling an Internet of Value. It's impossible to list every project, tool, and business that interacts with the XRP Ledger, so this page only lists a few categories and highlights some central projects that are documented on this website.
+![The XRPL Ecosystem](img/ecosystem-apps-and-services.svg)
```

Comment:
Preferably you should leave a blank line before the image, or else Markdown treats it as a hard-wrapped continuation of the previous paragraph.

```suggestion

![The XRPL Ecosystem](img/ecosystem-apps-and-services.svg)
```